plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -257,22 +210,35 @@ class NewCreateServiceModalForm extends Component {
     return CreateServiceModalFormUtil.applyPatch(baseConfig, patch);
   }
 
-  getRootErrorMessage() {
-    const rootErrors = this.state.errorList.reduce(function (errors, error) {
-      if (error.path.length !== 0) {
-        return errors;
-      }
-
-      errors.push(<Alert>{error.message}</Alert>);
-
-      return errors;
-    }, []);
+  getRootErrors() {
+    const errors = this.props.errors.filter(function (error) {
+      return error.path.length === 0;
+    });
 
-    if (rootErrors.length === 0) {
+    if (errors.length === 0) {
       return null;
     }
 
-    return rootErrors;
+    const errorItems = errors.map((error, index) => {
+      const prefix = error.path.length ? `${error.path.join('.')}:` : '';
+
+      return (
+        <li key={index} className="short">
+          {`\u2022 ${prefix} ${error.message}`}
```

Comment:
At the time `cnvs` was kind of broken and the `ul` was shown outside of the line bounds. I don't know if this is fixed though..